src/server_manager/model/account.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import {CloudProviderId} from "./cloud";
+import {DigitalOceanServer} from "../web_app/digitalocean_app/digitalocean_server";
+import {ManagedServer} from "./server";
+
+export class AccountId {
```

Comment:
The point of ids is to establish identity. Used for equality tests and indexing/lookups. It's better if they are opaque strings that we don't dig into.

Let's make it an opaque string instead, and add a cloudProvider method to Account so you can group them by account.
The id should probably merge the cloud id with the id of the account in that cloud to ensure uniqueness, but the user shouldn't rely on that.